packages/file/lib/src/backends/memory/memory_random_access_file.dart

```diff
@@ -0,0 +1,377 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:math' show min;
+import 'dart:typed_data';
+
+import 'package:file/src/common.dart' as common;
+import 'package:file/src/io.dart' as io;
+
+import 'memory_file.dart';
+import 'node.dart';
+import 'utils.dart' as utils;
+
+// TODO(jamesderlin): Currently ignores file permissions.
```

Comment:
Can you file a bug for that and link it here?

Reply:
I'll file a bug.  I'll also just remove this TODO since having a top-level TODO probably isn't very useful.